### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Tymmmy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,8 +50,6 @@
     "typescript": "^5.4.2"
   },
   "engines": {
-    "pnpm": "^8.15.4",
     "node": ">=20.0.0"
-  },
-  "packageManager": "pnpm@8.15.5"
+  }
```
Comment (raducristianpopa) [inline package.json:54]: We should keep the `engines` and `packageManager` in `package.json`

File: README.md [hunk 2]
```diff
@@ -1 +1,72 @@
...
+
+Never heard of Interledger before, or you would like to learn more? Here are some good places to start:
+
+- [Good first issues](https://github.com/interledger/testnet/contribute)
+- [Interledger Explainer Video](https://twitter.com/Interledger/status/1567916000074678272)
+- [Interledger Website](https://interledger.org)
+- [Payment pointers](https://paymentpointers.org/)
+- [Web monetization](https://webmonetization.org/)
+- [Interledger Test Wallet](https://rafiki.money)
+- [Interledger Boutique](https://rafiki.boutique)c
```
Comment (raducristianpopa) [inline README.md:23]:
```suggestion
- [Interledger Boutique](https://rafiki.boutique)
```

**Review by raducristianpopa — COMMENTED** [review]
Let's add a`.env.example` file with default values. For `PRIVATE_KEY`, `KEY_ID` and `WALLET_ADDRESS` we should have a small explanatory section.

File: package.json (hunk 1 above)
Comment (Tymmmy) [reply package.json:54]: We talked with Tadej, he added it there by mistake and all builds fail. In release version pnpm uses latest. So this should be removed

**Review by lengyel-arpad85 — APPROVED** [review]

**Pull request merged**